mopidy/audio/actor.py

```diff
@@ -229,24 +194,16 @@ def get_volume(self):
 
     def set_volume(self, volume):
         self._element.set_property('volume', volume / 100.0)
+        self._mixer.trigger_volume_changed(volume)
 
     def get_mute(self):
         return self._element.get_property('mute')
 
     def set_mute(self, mute):
-        return self._element.set_property('mute', bool(mute))
-
-    def _volume_changed(self, element, property_):
-        old_volume, self._last_volume = self._last_volume, self.get_volume()
-        if old_volume != self._last_volume:
-            gst_logger.debug('Notify volume: %s', self._last_volume / 100.0)
-            self._mixer.trigger_volume_changed(self._last_volume)
-
-    def _mute_changed(self, element, property_):
-        old_mute, self._last_mute = self._last_mute, self.get_mute()
-        if old_mute != self._last_mute:
-            gst_logger.debug('Notify mute: %s', self._last_mute)
-            self._mixer.trigger_mute_changed(self._last_mute)
+        result = self._element.set_property('mute', bool(mute))
+        if result:
```

Comment:
Because that is what the old code does, I don't know why we check for mute, but not for volume.